i18n-scripts/lexers.js

```diff
@@ -0,0 +1,30 @@
+const EventEmitter = require('events');
```

Comment:
We should consider moving this into the template.

Reply:
not sure what template you mean?

Reply:
openshift/console-plugin-template

Reply:
will create a story for that